xdcs-frontend/src/app/element/log-preview/log-preview.component.ts

```diff
@@ -35,4 +36,5 @@ export interface LogLine {
   lineNumber: number;
   time: Date;
   contents: string;
+  agentId: String;
```

Comment:
to nie jest ID agenta tylko ID node'a

Reply:
poza tym na poziomie `log-preview` to miało być nazwane tagiem, tak jak rozmawialiśmy